features/enlinkd/adapters/common/src/main/java/org/opennms/netmgt/enlinkd/TopologyUpdater.java

```diff
@@ -103,8 +102,8 @@ public void register() {
             m_topologyDao.register(this);
             m_registered  = true;
             LOG.info("register: protocol:{}", getProtocol().getId());
-        } catch (OnmsTopologyException e) {
-            LOG.error("register: protocol:{} {}", e.getProtocol(),e.getMessage());
+        } catch (Exception e) {
+            LOG.error("register: {}", e.getMessage());
```

Comment:
is it ok to loose the stack trace? Alternative: LOG.error("register: {}", e);

Reply:
You are right, we shouldn't lose the stack trace, I'll update these to output the trace in the log message.